pkg/plugins/golang/v3/api.go

```diff
@@ -107,30 +104,32 @@ After the scaffold is written, api will run make on the project.
 }
 
 func (p *createAPISubcommand) BindFlags(fs *pflag.FlagSet) {
-	fs.BoolVar(&p.runMake, "make", true, "if true, run make after generating files")
+	p.options = &model.Options{Domain: p.config.Domain}
+	fs.StringVar(&p.options.Group, "group", "", "resource Group")
+	// TODO: allow the user to specify the domain
+	fs.StringVar(&p.options.Version, "version", "", "resource Version")
+	fs.StringVar(&p.options.Kind, "kind", "", "resource Kind")
 
-	fs.BoolVar(&p.doResource, "resource", true,
+	fs.BoolVar(&p.options.DoDefinition, "resource", true,
```

Comment:
DoDefinition? What it means?

Reply:
Oh, I forgot to change this. It is the old `doResource` but it should be `DoAPI` not `DoDefinition`.